### PR TITLE
{bio}[foss/2016a] SNAPE-pooled r32 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyconfigs/s/SNAPE-pooled/SNAPE-pooled-r32_fix-hardcoding-Makefile.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+fix hardcoded 'gcc' + add optimization flags through $CFLAGS
+author: Kenneth Hoste (HPC-UGent)
+--- snape-pooled/trunk/Makefile.orig	2016-03-10 10:23:16.000000000 +0100
++++ snape-pooled/trunk/Makefile	2016-03-10 10:23:32.000000000 +0100
+@@ -8,6 +8,6 @@
+ 	latex $<
+ 	dvips -o snape-pooled-manual.ps snape-pooled-manual.dvi
+ snape-pooled-fast: snape-pooled.c
+-	gcc -g -o $@ $< -lm
++	${CC} ${CFLAGS} -g -o $@ $< -lm
```
Comment (boegel) [inline easybuild/easyconfigs/s/SNAPE-pooled/SNAPE-pooled-r32_fix-hardcoding-Makefile.patch]: I like how they call the binary 'fast', yet they don't use any optimisation flags (and thus they default to `-O0` with GCC)...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2715.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3db6c2ce994f831ccc98 for a full test report.

**Comment by wpoely86** [comment]
:+1:

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2074.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2780ec56a3a5393f4508 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**